### PR TITLE
Set UI styling for MAK exhibition according to designer's suggestions

### PR DESCRIPTION
**Pull request by janper**

### CODE REVIEW

File: src/ui.rs [hunk 1]
```diff
@@ -324,6 +343,11 @@ impl<'a> UiFrame<'a> {
     ) -> bool {
         let ui = &self.imgui_ui;
 
+        let window_style = ui.push_style_color(
```
Comment (yanchith) [inline src/ui.rs]: Usually called `color_token` elsewhere

File: src/ui.rs [hunk 2]
```diff
@@ -810,6 +845,7 @@ impl<'a> UiFrame<'a> {
 
                             if ui.is_item_hovered() {
                                 if let Some(error) = error {
+                                    let tooltip_bgcolor_token = ui.push_style_color(imgui::StyleColor::PopupBg, self.colors.header_error_hovered);
```
Comment (yanchith) [inline src/ui.rs]: Usually called `color_token`

File: src/ui.rs [hunk 3]
```diff
@@ -1270,6 +1309,7 @@ impl<'a> UiFrame<'a> {
                         imgui::StyleColor::ButtonHovered,
                         self.colors.special_button_hovered,
                     ),
+                    (imgui::StyleColor::TextDisabled, self.colors.special_button_text),
```
Comment (yanchith) [inline src/ui.rs]: Please sort this below "text" and above "button"

File: src/ui.rs [hunk 4]
```diff
@@ -118,11 +121,14 @@ impl Ui {
             combo_box_selected_item: style[imgui::StyleColor::Header],
             combo_box_selected_item_hovered: style[imgui::StyleColor::HeaderHovered],
             combo_box_selected_item_active: style[imgui::StyleColor::HeaderActive],
-            log_message_info: [0.70, 0.70, 0.70, 1.0],
+            log_message_info: [0.3, 0.3, 0.3, 1.0],
             log_message_warn: [0.80, 0.80, 0.05, 1.0],
             log_message_error: [1.0, 0.15, 0.05, 1.0],
             header_error: [0.85, 0.15, 0.05, 0.4],
             header_error_hovered: [1.00, 0.15, 0.05, 0.4],
+            tooltip_text: [1.0, 1.0, 1.0, 1.0],
```
Comment (yanchith) [inline src/ui.rs:143]: No need for this to exist if you always set it to white for both themes. Instead can be  `const COLOR_TOOLPTIP_TEXT`, defined somewhere at the top of the file.

File: src/ui.rs [hunk 5]
```diff
@@ -416,10 +448,8 @@ impl<'a> UiFrame<'a> {
         let notifications_window_vertical_position =
             MARGIN * 2.0 + (1.0 - NOTIFICATIONS_WINDOW_HEIGHT_MULT) * window_inner_height;
 
-        let color_token = ui.push_style_colors(&[
-            (imgui::StyleColor::Border, [0.0, 0.0, 0.0, 0.1]),
```
Comment (yanchith) [inline src/ui.rs:434]: This removal on purpose? :cry:

**Review by yanchith — COMMENTED** [review]
Nice

File: src/ui.rs (hunk 4 above)
Comment (janper) [reply src/ui.rs:143]: I'm not sure this remains the same for both themes. Also I prefer keeping setting regarding one thing together - it's already difficult enough to look for color setting all over the file.

File: src/ui.rs (hunk 5 above)
Comment (janper) [reply src/ui.rs:434]: absolutely

**Comment by janper** [comment]
One question: couldn't find where is the overlay that greys-out the UI when a modal (screenshot) window is displayed. The grey overlay is barely visible in the funky theme.

**Comment by yanchith** [comment]
> One question: couldn't find where is the overlay that greys-out the UI when a modal (screenshot) window is displayed. The grey overlay is barely visible in the funky theme.

Don't know where and if this is configurable.

File: src/ui.rs (hunk 1 above)
Comment (yanchith) [reply src/ui.rs]: You probably forgot to do this one

File: src/ui.rs (hunk 3 above)
Comment (yanchith) [reply src/ui.rs]: Meh, not there.

Comment (janper) [reply src/ui.rs]: Not sure I understand. I did something though.

File: src/ui.rs (hunk 4 above)
Comment (yanchith) [reply src/ui.rs:143]: Ok, not a biggie.

**Review by yanchith — APPROVED** [review]

**Pull request merged**